testcases/multizone_large_office_eplus/doc/index.html

```diff
@@ -0,0 +1,317 @@
+<html>
+<h3>General</h3>
+<p>
+This is a multi zone large office EnergyPlus model. It is based on DOE Commercial Reference Large Office 
+Building model, post-1980 construction, ASHRAE Standards 90.1-1989 and 62-1999.
+The original model has 12 floors (10 identical middle floors) with a basement. Due to the current limitation
+of Spawn for the floor multiplier, this emulator only contains three different floors (i.e., top, middle, bottom).
+The ground floor assumes adiabatic with the basement. 
+</p>
+
+</p>
+<p align=\"center\">
+<img src="images/buildingshape.png" alt=\"image\"/>
+<figcaption><small>Figure 1: Building Shape.</small></figcaption>
+</p>
+
+<h3>Building Design and Use</h3>
+<h4>Architecture</h4>
+<p>
+Number of Stories = 3; Shape = rectangle, Aspect ratio = 1.5.
+</p>
+
+<h4>Constructions</h4>
+<p>
+Envelope thermal properties vary with climate according to ASHRAE Standard 90.1-1989.
+Opaque constructions: mass walls; built-up flat roof (insulation above deck); slab-on-grade floor
+Windows: window-to-wall ratio = 38.0%, equal distribution of windows
+Infiltration in perimeter zones only
+= 1.5 cfm/ft2 above grade wall area at 0.3 in wc (75 Pa) adjusted to 0.016 in wc (4 Pa).
+25% of full value when ventilation system on.
+</p>
+<h4>Occupancy schedules</h4>
+<p>
+See BLDG_OCC_SCH in attached idf.
+</p>
+
+
+<h4>Internal loads and schedules</h4>
+<p>
+See BLDG_LIGHT_SCH, BLDG_EQUIP_SCH, BLDG_ELEVATORS in attached idf.
+</p>
+
+
+<h4>Climate data</h4>
+<p>
+The weather data is from TMY3 for Chicago O'Hare International Airport.
+</p>
+<h3>HVAC System Design</h3>
+
+<p>
+The HVAC system are ideal load systems, which calculates the sensible/latent cooling/heating load.
```

Comment:
It is worth mentioning more about the ideal load systems. Like, what are the assumptions?